google/cloud/security/inventory/inventory_loader.py

```diff
@@ -124,25 +110,25 @@ def _exists_snapshot_cycles_table(dao):
 
     return False
 
-def _create_snapshot_cycles_table(dao):
+def _create_snapshot_cycles_table(dao_):
```

Comment:
This is to fix lint's complaint about using "dao" locally here, would rewrite the global "dao" from the import.  Let me change this to "inventory_dao" to be consistent with the flags.  (same problem there, too; local "flags" would overwrite the imported "flags")